subsys/usb/usb_device.c

```diff
@@ -931,6 +933,19 @@ static void usb_register_status_callback(usb_dc_status_callback cb)
 	usb_dev.status_callback = cb;
 }
 
+/*
+ * @brief register user callback for device status
+ *
+ * This function registers a callback for device status. The registered callback
+ * is used to report changes in the status of the device controller to APP.
+ *
+ * @param [in] cb Callback function pointer
+ */
+static void usb_register_user_status_callback(usb_dc_status_callback cb)
```

Comment:
I was following the existing style it is present to this file. Same was done to [usb_register_status_callback](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/usb/usb_device.c#L929). I wanted to be consistent but of course it could be simplified, I do not insist.

Reply:
Please simplify :-)